docs/dev/projectsetup/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Project Build Setup"
```

Comment:
Yes, I had `Project Setup` first, but added `Build` to highlight that this section is about configuring the build file (`pom.xml` in case of Maven) for a Flink application. Project in this context refers to the application project but it is not really distinguishing (could also be the Flink project).

Reply:
Sounds reasonable :-)